src/index.html

```diff
@@ -4,10 +4,56 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="page">
+      <div class="card" data-qa="card">
+        <div class="card__item item">
+
+          <div class="item__image-container">
+            <img class="item__image" src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock">
+          </div>
+
+          <h2 class="item__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="item__code">Product code: 195434</p>
+
+          <div class="item__rate-container rate">
+            <div class="rate__stars-container stars rate__stars-container_stars--1">
+              <ul class="list stars__list">
+                <li class="list__item"></li>
+                <li class="list__item"></li>
+                <li class="list__item"></li>
+                <li class="list__item"></li>
+                <li class="list__item"></li>
+              </ul>
+            </div>
+            <div class="rate__review review">
+              <p class="review__item">Reviews: 5</p>
+            </div>
+          </div>
+
+          <div class="item__price-container price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
+
+          <button class="item__button" type="submit">
+            <a class="item__link" href="/productLink"
```

Comment:
Formatting is really weird here. Please use the autoformatter

Reply:
> Formatting is really weird here. Please use the autoformatter

I use htmlint and editor config with 2 spaces, can you explaine me how it should be?

Reply:
> Formatting is really weird here. Please use the autoformatter

Fixed